inc/saiswitch.h

```diff
@@ -2834,6 +2834,22 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_SWITCH_ISOLATE,
 
+    /**
+     * @brief Available IP Next hop group member entries
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_IP_NEXT_HOP_GROUP_MEMBER_ENTRY,
+
+    /**
+     * @brief Available number of VOQ in the VOQ System (chassis)
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_SYSTEM_VOQS,
```

Comment:
Do we still need this attribute? vs just get this info like so
``
sai_attribute_t attr[1];
attr[0].id = SAI_QUEUE_ATTR_TYPE;
attr[0].value = SAI_QUEUE_TYPE_UNICAST_VOQ; // or SAI_QUEUE_TYPE_MULTICAST_VOQ
sai_object_type_get_availability(<switch_id>, SAI_OBJECT_TYPE_QUEUE, 1,
attr, &count)

Reply:
There are two kind of VoQ architecture.
1. Where there is dedicated unicast and multicast voqs. In this case existing attribute SAI_QUEUE_TYPE_UNICAST_VOQ and SAI_QUEUE_TYPE_MULTICAST_VOQ should be queried.
2. Where is a common VoQ pool and dynamically unicast and multicast VOQs are allocated. In this case new attribute SAI_SWITCH_ATTR_AVAILABLE_SYSTEM_VOQS should be queried.